javascript/transform.js

```diff
@@ -102,4 +109,44 @@ function distance(latA, lngA, latB, lngB) {
 	var distance = alpha * earthR;
 	return distance;
 }
-module.exports.distance = distance;
\ No newline at end of file
+exports.distance = distance;
+
+function gcj2bd(gcjLat, gcjLng) {
+	if (outOfChina(gcjLat, gcjLng)) {
+		return {"lat": gcjLat, "lng": gcjLng};
+	}
+
+	var x = gcjLng, y = gcjLat;
+	var z = Math.sqrt(x * x + y * y) + 0.00002 * Math.sin(y * Math.PI);
+	var theta = Math.atan2(y, x) + 0.000003 * Math.cos(x * Math.PI);
+	var bdLng = z * Math.cos(theta) + 0.0065;
+	var bgLat = z * Math.sin(theta) + 0.006;
```

Comment:
You're right, fix it in https://github.com/googollee/eviltransform/pull/14, thanks